test/e2e/cypress/e2e/saptune_details.cy.js

```diff
@@ -0,0 +1,113 @@
+describe('Saptune Details page', () => {
+  beforeEach(() => {
+    cy.visit('/hosts/13e8c25c-3180-5a9a-95c8-51ec38e50cfc/saptune');
+  });
+
+  const notFoundContainerSelector = '.pb-24';
+  const versionContainerSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(3)';
+  const tuningStatusIcon = ':nth-child(3) > :nth-child(2) > :nth-child(1)';
+  const saptuneServiceStatusSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(5)';
+  const saptuneServiceStatusIcon =
+    '.grid-flow-col > :nth-child(1) > :nth-child(2) > :nth-child(1) > .flex';
+  const sapconfServiceStatusIcon =
+    '.grid-flow-col > :nth-child(2) > :nth-child(2) > :nth-child(1) > .flex';
+  const tunedServiceStatusIcon =
+    ':nth-child(3) > :nth-child(2) > :nth-child(1) > .flex';
+  const saptuneTuningSolutionsSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(7)';
+  const saptuneTuningNotesSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(9)';
+  const saptuneStagingStatusSelector = ':nth-child(11)';
+
+  it('should render default not found if saptune payload status is null', () => {
+    cy.loadScenario('host-vmhdbdev01-saptune-unsupported');
+    cy.get(notFoundContainerSelector).should(
+      'contain',
+      'Saptune Details Not Found'
+    );
+  });
+
+  it('should render default not found if saptune payload is not installed', () => {
+    cy.loadScenario('host-vmhdbdev01-saptune-uninstalled');
+    cy.get(notFoundContainerSelector).should(
+      'contain',
+      'Saptune Details Not Found'
+    );
+  });
+
+  it('should render versions list view when saptune is not tuning correctly', () => {
+    cy.loadScenario('host-vmhdbdev01-saptune-not-tuned');
+    cy.get(versionContainerSelector)
```

Comment:
What about the `cy.get('div').contains('Package').next.should('contain', '3.1.0')` kind of style?
In this cases, testing hardcoded texts like `Package` doesn't add much value anyway, as it is tested in UT